lib/backup/s3remote/s3.go

```diff
@@ -192,6 +227,11 @@ func (fs *FS) CopyPart(srcFS common.OriginFS, p common.Part) error {
 		CopySource: aws.String(copySource),
 		Key:        aws.String(dstPath),
 	}
+
+	if fs.StorageClass != emptyStorageClass {
```

Comment:
Are these checks necessary here?
Seems like `StorageClass` field is a wrapper type for `string`:
```golang
StorageClass StorageClass
...
type StorageClass string

```
So by default it should be also an empty string, and it will be safe to use default value of `""`.
Could you please check if this is correct?

Reply:
You are correct. the AWS code checks for length of the StorageClass field rather than using empty string comparison so I've replicated that in the validation function & removed the conditionals